apps/src/code-studio/components/AssetManager.jsx

```diff
@@ -40,7 +42,8 @@ var AssetManager = React.createClass({
   },
 
   componentWillMount: function () {
-    assetsApi.ajax('GET', '', this.onAssetListReceived, this.onAssetListFailure);
+    let api = this.props.useFilesApi ? filesApi : assetsApi;
+    api.ajax('GET', '', this.onAssetListReceived, this.onAssetListFailure);
```

Comment:
Seems like we should use some sort of `api.list(success, failure)` method instead of just `api.ajax('GET', '', success, failure)`